js/bybit.js

```diff
@@ -2782,20 +3033,20 @@ module.exports = class bybit extends Exchange {
         const request = {
             'symbol': market['id'],
             'side': this.capitalize (side),
-            'type': upperCaseType, // limit, market or limit_maker
+            'orderType': upperCaseType, // limit, market or limit_maker
             'timeInForce': 'GTC', // FOK, IOC
-            'qty': this.amountToPrecision (symbol, amount),
+            'orderQty': this.amountToPrecision (symbol, amount),
             // 'orderLinkId': 'string', // unique client order id, max 36 characters
         };
         if ((upperCaseType === 'LIMIT') || (upperCaseType === 'LIMIT_MAKER')) {
             if (price === undefined) {
                 throw new InvalidOrder (this.id + ' createOrder requires a price argument for a ' + type + ' order');
             }
-            request['price'] = parseFloat (this.priceToPrecision (symbol, price));
+            request['orderPrice'] = parseFloat (this.priceToPrecision (symbol, price));
```

Comment:
![image](https://user-images.githubusercontent.com/43336371/199766328-0d88d213-8418-424c-bf3d-304612e3113c.png)
does it work with strings as documented?

Reply:
As I remember, I have tested all those methods before pushing, but anyway, I've amended that to string now and re-tested and works with strings as API doc says.